.github/workflows/php-lint-test.yml

```
@@ -5,7 +5,7 @@ on:
 
 env:
   WP_VERSION:        latest
-  WC_L2_VERSION: '7.0.1'  # the min supported version as per L-2 policy
+  WC_L2_VERSION: '7.1.0'  # the min supported version as per L-2 policy
```

The newest WC `7.1` version is `7.1.1`

```suggestion
  WC_L2_VERSION: '7.1.1'  # the min supported version as per L-2 policy
```

---

Good call! Fixed in [d6a752b](https://github.com/Automattic/woocommerce-payments/pull/5336/commits/d6a752b6bcb83fd5932a16c62835e57d3ea172e8)